website/snippets/_adapters-trusted.md

```diff
@@ -5,6 +5,11 @@
     body="<a href='/docs/core/connect-data-platform/athena-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-athena-community><img src=https://badge.fury.io/py/dbt-athena-community.svg/></a>"
     icon="athena"/>
 
+<Card
+    title="Glue*"
+    body="<a href='/docs/core/connect-data-platform/athena-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-glue><img src=https://badge.fury.io/py/dbt-glue.svg/></a>"
```

Comment:
@dataders is the athena link the right one? or should it be the glue-setup one?

```suggestion
    body="<a href='/docs/core/connect-data-platform/glue-setup'><img src='/img/icons/dbt-bit.svg' width='7%'/>Install with dbt Core </a> <br /><br /><a href=https://badge.fury.io/py/dbt-glue><img src=https://badge.fury.io/py/dbt-glue.svg/></a>"
```

Reply:
good catch!!